server/src/frontend_api.rs

```diff
@@ -168,7 +168,7 @@ fn post_all_features(
     let engine = engine_cache.get(&key).ok_or_else(|| {
         EdgeError::FrontendNotYetHydrated(FrontendHydrationMissing::from(&edge_token))
     })?;
-    let feature_results = engine.resolve_all(&context_with_ip).unwrap();
+    let feature_results = engine.resolve_all(&context_with_ip, &None).unwrap();
```

Comment:
All of these changes are just to deal with the API change in Yggdrasil. This is the hook for custom strategies, that we're explicitly ignoring